fw/beef.h

```diff
@@ -79,6 +79,7 @@ void CALLBACK_HID_Device_ProcessHIDReport(USB_ClassInfo_HID_Device_t* const HIDI
 #define BUTTON_9 1 << 8
 #define BUTTON_10 1 << 9
 #define BUTTON_11 1 << 10
+#define EFFECTORS_ALL (BUTTON_8 | BUTTON_9 | BUTTON_10 | BUTTON_11)
```

Comment:
I don't think this is an appropriate name considering it includes non-effector buttons such as start, but to be honest I don't really have a better suggestion. We can leave it as-is for now.